ci/test-stable.sh

```diff
@@ -33,7 +33,7 @@ test-stable)
   echo "Executing $testName"
 
   _ cargo +"$rust_stable" build --all --tests --bins ${V:+--verbose}
-  _ cargo +"$rust_stable" test --all ${V:+--verbose} -- --nocapture
+  _ cargo +"$rust_stable" test --all ${V:+--verbose} --release -- --nocapture
```

Comment:
This is what I was trying to avoid. Can you filter out any module that uses local cluster from here? And then run all the ones that use local cluster separately with the release flag? Don't want to build all tests in debug.

Reply:
Also you need to update this for stable perf

Reply:
stable-perf doesn't run local cluster so I didn't change it.

Reply:
oh it doesnt?